src/assets/src/utils.ts

```diff
@@ -18,6 +19,24 @@ export const redirectToSearch = (term: string) => {
     location.href = `/search/${term}/?redirected=true`;
 }
 
+export const redirectToBackendAuth = (backend: string) => {
+    ReactGA.event({
+        category: "Auth",
+        action: `Redirected to ${backend} Auth`,
+        nonInteraction: true,
+    });
+    location.href = `/auth/${backend}/?state=${location.pathname}`;
+}
+
+export const checkBackendAuth = (user: MyUser, queue: QueueHost) => {
+    for (const backend of queue.allowed_backends) {
+        const authorized = user.authorized_backends[backend];
+        if (authorized === false) {
```

Comment:
Nit-pick: This could be simplified to `!authorized`?

Reply:
Depends what kind of behavior we want in the event queues are created with a backend, and then the backend is disabled for the application instance. This code would then get `const authorized = undefined`. If using `!authorized`, you get redirected to an error page every time you visit the queue. As I currently have it, the redirect is skipped, and the page might act funny in other ways, but at least you can get a better idea of what's going on.

Reply:
Oh, I missed some nuance here. We can keep this as you have it, though it may be worth talking about this edge case more in the future.